src/videos.md

```diff
@@ -0,0 +1,258 @@
+# MFEM Videos
+
+A collection of MFEM-related videos, including recorded talks from the MFEM workshops and conference presentations.
+
+## MFEM Workshop 2021
```

Comment:
Do we want to add these videos to the workshop page as well?

Reply:
@hauten updated the workshop page with links to them. I think this is better than embedding them there, because otherwise the page will get overwhelmed with the videos (they need to be relatively big)

Reply:
Oh I see the links now.  Thanks @hauten